### PR TITLE
(WIP) sbt-plugin changes for sbt-cross-project

### PR DESCRIPTION
**Pull request by MasseGuillaume**
https://github.com/scala-native/sbt-cross-project/pull/3

### CODE REVIEW

File: docs/user/sbt.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Start within a new folder, and create a file ``project/plugins.sbt`` as follows:
 
     resolvers += Resolver.sonatypeRepo("snapshots")
 
-    addSbtPlugin("org.scala-native" % "sbtplugin"  % "0.1-SNAPSHOT")
+    addSbtPlugin("org.scala-native" % "sbtplugin"  % "0.1.0-SNAPSHOT")
```
Comment (MasseGuillaume) [inline docs/user/sbt.rst]: http://semver.org/#summary

Comment (MasseGuillaume) [inline docs/user/sbt.rst]: hum there is something strange in the sbt build. it does not compile if I set the version to 0.1.0-SNAPSHOT. _If it's not broken dont fix it_

**Comment by MasseGuillaume** [comment]
Travis is weird ^, it should work fine.

**Comment by densh** [comment]
Try `bin/scalafmt --version 0.4.7`.

**Comment by MasseGuillaume** [comment]
as discussed I have to split this PR

* [ ] nixos support improvement
* [ ] %%% (cross compiled artifact)
* [ ] runtime bootstrap caching aware of clang options
* [ ] misc changes (format bin, etc)

**Comment by densh** [comment]
Is this PR up-to-date with our latest discussion on cross projects?

**Comment by MasseGuillaume** [comment]
not yet

**Comment by densh** [comment]
Lets close it for now then, given that it's both failing the build and out of date. Feel free open a new one once it's ready.

**Pull request closed without merging**